// MIT Â© 2017 azu
import { GitHubSearchQuery } from "../../domain/GitHubSearch/GitHubSearchList/GitHubSearchQuery";
import { GitHubSearchResult } from "../../domain/GitHubSearch/GitHubSearchStream/GitHubSearchResult";
import { GitHubSearchResultFactory } from "../../domain/GitHubSearch/GitHubSearchStream/GitHubSearchResultFactory";
import { GitHubSetting } from "../../domain/GitHubSetting/GitHubSetting";
import { Item } from "../../domain/GitHubSearch/GitHubSearchStream/GitHubSearchResultItem";

const sample = {
    "total_count": 3344,
    "incomplete_results": false,
    "items": [
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/3408",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/3408/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/3408/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/3408/events",
            "html_url": "https://github.com/sympy/sympy/issues/3408",
            "id": 28948494,
            "number": 3408,
            "title": "plotting on macos x freezes",
            "user": {
                "login": "certik",
                "id": 20568,
                "avatar_url": "https://avatars0.githubusercontent.com/u/20568?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/certik",
                "html_url": "https://github.com/certik",
                "followers_url": "https://api.github.com/users/certik/followers",
                "following_url": "https://api.github.com/users/certik/following{/other_user}",
                "gists_url": "https://api.github.com/users/certik/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/certik/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/certik/subscriptions",
                "organizations_url": "https://api.github.com/users/certik/orgs",
                "repos_url": "https://api.github.com/users/certik/repos",
                "events_url": "https://api.github.com/users/certik/events{/privacy}",
                "received_events_url": "https://api.github.com/users/certik/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 84627346,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/plotting",
                    "name": "plotting",
                    "color": "0000FF",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 17,
            "created_at": "2007-08-20T05:37:18Z",
            "updated_at": "2016-03-07T03:34:54Z",
            "closed_at": null,
            "body": "bc.. I don't have a macosx, but when I was presenting at SciPy2007, I borrowed a\nMac OS X laptop and sympy was working well (with python2.5), except plotting:\n\nPlot(x*y)\n\nworked fine, then I did:\n\nPlot(sin(x)*cos(y))\n\nand the plotting window stopped responding (the \"hour\" cursor when moved\nover it with mouse). I think it happened always when I called Plot for the\nsecond time.\n\nIt would be nice, if someone with macosx could verify that && fix that.\n\np. Original issue for \"#3408\":https://github.com/sympy/sympy/issues/3408: \"http://code.google.com/p/sympy/issues/detail?id=309\":http://code.google.com/p/sympy/issues/detail?id=309\n\np. Original author: \"https://code.google.com/u/104039945248245758823/\":https://code.google.com/u/104039945248245758823/\n\np. Original owner: \"https://code.google.com/u/115549613129666820104/\":https://code.google.com/u/115549613129666820104/\n",
            "score": 0.14507581
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/3552",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/3552/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/3552/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/3552/events",
            "html_url": "https://github.com/sympy/sympy/issues/3552",
            "id": 28948669,
            "number": 3552,
            "title": "ironpython: AttributeError: 'dictproxy' object has no attribute 'get'",
            "user": {
                "login": "sympy-issue-migrator",
                "id": 6188475,
                "avatar_url": "https://avatars3.githubusercontent.com/u/6188475?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/sympy-issue-migrator",
                "html_url": "https://github.com/sympy-issue-migrator",
                "followers_url": "https://api.github.com/users/sympy-issue-migrator/followers",
                "following_url": "https://api.github.com/users/sympy-issue-migrator/following{/other_user}",
                "gists_url": "https://api.github.com/users/sympy-issue-migrator/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/sympy-issue-migrator/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/sympy-issue-migrator/subscriptions",
                "organizations_url": "https://api.github.com/users/sympy-issue-migrator/orgs",
                "repos_url": "https://api.github.com/users/sympy-issue-migrator/repos",
                "events_url": "https://api.github.com/users/sympy-issue-migrator/events{/privacy}",
                "received_events_url": "https://api.github.com/users/sympy-issue-migrator/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627441,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/Alternate%20Python",
                    "name": "Alternate Python",
                    "color": "800080",
                    "default": false
                },
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 11,
            "created_at": "2007-11-18T04:48:18Z",
            "updated_at": "2015-04-17T16:58:11Z",
            "closed_at": null,
            "body": "bc.. Trying to use with IRONPython and get the error below:\n\n  File C:\\Python24\\Lib\\site-packages\\sympy__init__.py, line 4, in \nInitialize\n  File , line 0, in _stub_##2\n  File C:\\Python24\\Lib\\site-packages\\sympy\\core__init__.py, line 26, in \nInitialize\nAttributeError: 'dictproxy' object has no attribute 'get'\n\nAny thoughts?\n\np. Original issue for \"#3552\":https://github.com/sympy/sympy/issues/3552: \"http://code.google.com/p/sympy/issues/detail?id=453\":http://code.google.com/p/sympy/issues/detail?id=453\n\np. Original author: \"https://code.google.com/u/115961875564935506885/\":https://code.google.com/u/115961875564935506885/\n",
            "score": 0.3513895
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/3791",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/3791/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/3791/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/3791/events",
            "html_url": "https://github.com/sympy/sympy/issues/3791",
            "id": 28948943,
            "number": 3791,
            "title": "copy (not move) isympy to isympy.py",
            "user": {
                "login": "certik",
                "id": 20568,
                "avatar_url": "https://avatars0.githubusercontent.com/u/20568?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/certik",
                "html_url": "https://github.com/certik",
                "followers_url": "https://api.github.com/users/certik/followers",
                "following_url": "https://api.github.com/users/certik/following{/other_user}",
                "gists_url": "https://api.github.com/users/certik/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/certik/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/certik/subscriptions",
                "organizations_url": "https://api.github.com/users/certik/orgs",
                "repos_url": "https://api.github.com/users/certik/repos",
                "events_url": "https://api.github.com/users/certik/events{/privacy}",
                "received_events_url": "https://api.github.com/users/certik/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": {
                "login": "certik",
                "id": 20568,
                "avatar_url": "https://avatars0.githubusercontent.com/u/20568?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/certik",
                "html_url": "https://github.com/certik",
                "followers_url": "https://api.github.com/users/certik/followers",
                "following_url": "https://api.github.com/users/certik/following{/other_user}",
                "gists_url": "https://api.github.com/users/certik/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/certik/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/certik/subscriptions",
                "organizations_url": "https://api.github.com/users/certik/orgs",
                "repos_url": "https://api.github.com/users/certik/repos",
                "events_url": "https://api.github.com/users/certik/events{/privacy}",
                "received_events_url": "https://api.github.com/users/certik/received_events",
                "type": "User",
                "site_admin": false
            },
            "assignees": [
                {
                    "login": "certik",
                    "id": 20568,
                    "avatar_url": "https://avatars0.githubusercontent.com/u/20568?v=3",
                    "gravatar_id": "",
                    "url": "https://api.github.com/users/certik",
                    "html_url": "https://github.com/certik",
                    "followers_url": "https://api.github.com/users/certik/followers",
                    "following_url": "https://api.github.com/users/certik/following{/other_user}",
                    "gists_url": "https://api.github.com/users/certik/gists{/gist_id}",
                    "starred_url": "https://api.github.com/users/certik/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/certik/subscriptions",
                    "organizations_url": "https://api.github.com/users/certik/orgs",
                    "repos_url": "https://api.github.com/users/certik/repos",
                    "events_url": "https://api.github.com/users/certik/events{/privacy}",
                    "received_events_url": "https://api.github.com/users/certik/received_events",
                    "type": "User",
                    "site_admin": false
                }
            ],
            "milestone": null,
            "comments": 10,
            "created_at": "2008-02-10T10:39:45Z",
            "updated_at": "2015-04-17T16:59:21Z",
            "closed_at": null,
            "body": "bc.. > Perfect. Is there something you would add to our installation instructions?\n\n> I need to get to some windows machine and try the experience for myself.\n> Just one thing. Copy isympy.py in Scripts directory along with plain\n> isympy as windows can not executes files without an extension.\n\np. Original issue for \"#3791\":https://github.com/sympy/sympy/issues/3791: \"http://code.google.com/p/sympy/issues/detail?id=692\":http://code.google.com/p/sympy/issues/detail?id=692\n\np. Original author: \"https://code.google.com/u/104039945248245758823/\":https://code.google.com/u/104039945248245758823/\n\np. Original owner: \"https://code.google.com/u/104039945248245758823/\":https://code.google.com/u/104039945248245758823/\n",
            "score": 0.572412
        },
        {
            "url": "https://api.github.com/repos/fredrik-johansson/mpmath/issues/77",
            "repository_url": "https://api.github.com/repos/fredrik-johansson/mpmath",
            "labels_url": "https://api.github.com/repos/fredrik-johansson/mpmath/issues/77/labels{/name}",
            "comments_url": "https://api.github.com/repos/fredrik-johansson/mpmath/issues/77/comments",
            "events_url": "https://api.github.com/repos/fredrik-johansson/mpmath/issues/77/events",
            "html_url": "https://github.com/fredrik-johansson/mpmath/issues/77",
            "id": 32125397,
            "number": 77,
            "title": "setup.py lacks shebang",
            "user": {
                "login": "vks",
                "id": 33460,
                "avatar_url": "https://avatars1.githubusercontent.com/u/33460?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/vks",
                "html_url": "https://github.com/vks",
                "followers_url": "https://api.github.com/users/vks/followers",
                "following_url": "https://api.github.com/users/vks/following{/other_user}",
                "gists_url": "https://api.github.com/users/vks/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/vks/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/vks/subscriptions",
                "organizations_url": "https://api.github.com/users/vks/orgs",
                "repos_url": "https://api.github.com/users/vks/repos",
                "events_url": "https://api.github.com/users/vks/events{/privacy}",
                "received_events_url": "https://api.github.com/users/vks/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 94738783,
                    "url": "https://api.github.com/repos/fredrik-johansson/mpmath/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 94738780,
                    "url": "https://api.github.com/repos/fredrik-johansson/mpmath/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 94738781,
                    "url": "https://api.github.com/repos/fredrik-johansson/mpmath/labels/Priority-Medium",
                    "name": "Priority-Medium",
                    "color": "ededed",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 9,
            "created_at": "2008-04-24T01:23:02Z",
            "updated_at": "2014-04-24T07:14:21Z",
            "closed_at": null,
            "body": "bc.. $ ./setup.py install  \n./setup.py: line 1: from: command not found  \n: command not found  \n'/setup.py: line 3: syntax error near unexpected token `name='mpmath',\n'/setup.py: line 3:`setup(name='mpmath',  \n$ python setup.py install\nrunning install  \nrunning build  \nrunning build_py \n[...]                               \n\nThis is trivial, simple #!python in the first line would do the job.\n\np. Original issue for \"#77\":https://github.com/fredrik-johansson/mpmath/issues/77: \"http://code.google.com/p/mpmath/issues/detail?id=37\":http://code.google.com/p/mpmath/issues/detail?id=37\n\np. Original author: \"https://code.google.com/u/Vinzent.Steinberg@gmail.com/\":https://code.google.com/u/Vinzent.Steinberg@gmail.com/\n",
            "score": 0.25339288
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/3953",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/3953/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/3953/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/3953/events",
            "html_url": "https://github.com/sympy/sympy/issues/3953",
            "id": 28949129,
            "number": 3953,
            "title": "new lambdify fails to raise an exception",
            "user": {
                "login": "vks",
                "id": 33460,
                "avatar_url": "https://avatars1.githubusercontent.com/u/33460?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/vks",
                "html_url": "https://github.com/vks",
                "followers_url": "https://api.github.com/users/vks/followers",
                "following_url": "https://api.github.com/users/vks/following{/other_user}",
                "gists_url": "https://api.github.com/users/vks/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/vks/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/vks/subscriptions",
                "organizations_url": "https://api.github.com/users/vks/orgs",
                "repos_url": "https://api.github.com/users/vks/repos",
                "events_url": "https://api.github.com/users/vks/events{/privacy}",
                "received_events_url": "https://api.github.com/users/vks/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 15,
            "created_at": "2008-05-24T00:21:19Z",
            "updated_at": "2015-04-17T16:58:11Z",
            "closed_at": null,
            "body": "bc.. I was reviving my efforts to compile functions to machine code and got this\n(you will probably need the latest revision):\n\nTraceback (most recent call last):\n  File \"/home/one/src/sympy/compilef.py\", line 331, in <module>\n    benchmark(f)\n  File \"/home/one/src/sympy/compilef.py\", line 302, in benchmark\n    pf = lambdify(var, f)\n  File\n\"/usr/local/lib/python2.5/site-packages/sympy/utilities/lambdify.py\", line\n169, in lambdify\n    buf = _get_namespace(m)\n  File\n\"/usr/local/lib/python2.5/site-packages/sympy/utilities/lambdify.py\", line\n184, in _get_namespace\n    _import(m)\n  File\n\"/usr/local/lib/python2.5/site-packages/sympy/utilities/lambdify.py\", line\n86, in _import\n    raise ImportError, \"Can't import %s with command %s\"%(modulename,\nNameError: name 'modulename' is not defined\n\nIt seems to fail at raising an exception because uses a non-existent name.\nI think that it tries to analyze the namespace and fails due to being used\nfrom a function, but I'm not sure. I did not yet look into lambdify's\nimplementation and I still need to figure out what exactly causes the bug.\n\nAt least the NameError should be fixed.\n\np. Original issue for \"#3953\":https://github.com/sympy/sympy/issues/3953: \"http://code.google.com/p/sympy/issues/detail?id=854\":http://code.google.com/p/sympy/issues/detail?id=854\n\np. Original author: \"https://code.google.com/u/Vinzent.Steinberg@gmail.com/\":https://code.google.com/u/Vinzent.Steinberg@gmail.com/\n",
            "score": 0.107590616
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/3963",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/3963/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/3963/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/3963/events",
            "html_url": "https://github.com/sympy/sympy/issues/3963",
            "id": 28949140,
            "number": 3963,
            "title": "fix upgrading via setup.py or add a big fat warning",
            "user": {
                "login": "vks",
                "id": 33460,
                "avatar_url": "https://avatars1.githubusercontent.com/u/33460?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/vks",
                "html_url": "https://github.com/vks",
                "followers_url": "https://api.github.com/users/vks/followers",
                "following_url": "https://api.github.com/users/vks/following{/other_user}",
                "gists_url": "https://api.github.com/users/vks/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/vks/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/vks/subscriptions",
                "organizations_url": "https://api.github.com/users/vks/orgs",
                "repos_url": "https://api.github.com/users/vks/repos",
                "events_url": "https://api.github.com/users/vks/events{/privacy}",
                "received_events_url": "https://api.github.com/users/vks/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 24,
            "created_at": "2008-05-27T21:34:52Z",
            "updated_at": "2015-04-17T16:58:11Z",
            "closed_at": null,
            "body": "bc.. >>> from sympy import *\n Traceback (most recent call last):\n  File \"C:\\Dokumente und Einstellungen\\Vinzent\\Desktop\\newmax.py\", line 1,\nin <module>\n    from sympy import *\n  File \"C:\\Python25\\Lib\\site-packages\\sympy__init__.py\", line 22, in <module>\n    from sympy.core import *\n  File \"C:\\Python25\\Lib\\site-packages\\sympy\\core__init__.py\", line 4, in\n<module>\n    from basic import Basic, S, C, sympify\n  File \"C:\\Python25\\Lib\\site-packages\\sympy\\core\\basic.py\", line 5, in <module>\n    import sympy.thirdparty.mpmath as mpmath\n  File \"C:\\Python25\\Lib\\site-packages\\sympy\\thirdparty\\mpmath__init__.py\",\nline 5, in <module>\n    from mptypes import *\n  File \"C:\\Python25\\Lib\\site-packages\\sympy\\thirdparty\\mpmath\\mptypes.py\",\nline 17, in <module>\n    from libmpc import *\n  File \"C:\\Python25\\Lib\\site-packages\\sympy\\thirdparty\\mpmath\\libmpc.py\",\nline 19, in <module>\n    def mpc_add((a, b), (c, d), prec, rnd=round_fast):\nNameError: name 'round_fast' is not defined\n\nI installed latest revision via setup.py install on Windows. If this is due\nto not removing sympy before, setup.py has to be fixed.\n\np. Original issue for \"#3963\":https://github.com/sympy/sympy/issues/3963: \"http://code.google.com/p/sympy/issues/detail?id=864\":http://code.google.com/p/sympy/issues/detail?id=864\n\np. Original author: \"https://code.google.com/u/Vinzent.Steinberg@gmail.com/\":https://code.google.com/u/Vinzent.Steinberg@gmail.com/\n",
            "score": 0.26983762
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/3971",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/3971/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/3971/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/3971/events",
            "html_url": "https://github.com/sympy/sympy/issues/3971",
            "id": 28949151,
            "number": 3971,
            "title": "remove dependency on pexpect and add support for Mathtex",
            "user": {
                "login": "vks",
                "id": 33460,
                "avatar_url": "https://avatars1.githubusercontent.com/u/33460?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/vks",
                "html_url": "https://github.com/vks",
                "followers_url": "https://api.github.com/users/vks/followers",
                "following_url": "https://api.github.com/users/vks/following{/other_user}",
                "gists_url": "https://api.github.com/users/vks/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/vks/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/vks/subscriptions",
                "organizations_url": "https://api.github.com/users/vks/orgs",
                "repos_url": "https://api.github.com/users/vks/repos",
                "events_url": "https://api.github.com/users/vks/events{/privacy}",
                "received_events_url": "https://api.github.com/users/vks/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 23,
            "created_at": "2008-06-03T18:43:56Z",
            "updated_at": "2015-04-17T16:58:11Z",
            "closed_at": null,
            "body": "## bc.. In [39]: preview(x**x)\n\nImportError                               Traceback (most recent call last)\n\n/home/one/<ipython console> in <module>()\n\n/usr/local/lib/python2.5/site-packages/sympy-0.5.15-py2.5.egg/sympy/printing/preview.pyc\nin preview(expr, output, viewer, euler)\n     53     # we don't want to depend on anything not in the\n     54     # standard library with SymPy by default\n---> 55     import pexpect\n     56\n     57     special = [ 'pyglet' ]\n\nImportError: No module named pexpect\n\np. Original issue for \"#3971\":https://github.com/sympy/sympy/issues/3971: \"http://code.google.com/p/sympy/issues/detail?id=872\":http://code.google.com/p/sympy/issues/detail?id=872\n\np. Original author: \"https://code.google.com/u/Vinzent.Steinberg@gmail.com/\":https://code.google.com/u/Vinzent.Steinberg@gmail.com/\n",
            "score": 0.4166967
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/4216",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/4216/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/4216/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/4216/events",
            "html_url": "https://github.com/sympy/sympy/issues/4216",
            "id": 28949449,
            "number": 4216,
            "title": "Setup buildbot (again)",
            "user": {
                "login": "certik",
                "id": 20568,
                "avatar_url": "https://avatars0.githubusercontent.com/u/20568?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/certik",
                "html_url": "https://github.com/certik",
                "followers_url": "https://api.github.com/users/certik/followers",
                "following_url": "https://api.github.com/users/certik/following{/other_user}",
                "gists_url": "https://api.github.com/users/certik/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/certik/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/certik/subscriptions",
                "organizations_url": "https://api.github.com/users/certik/orgs",
                "repos_url": "https://api.github.com/users/certik/repos",
                "events_url": "https://api.github.com/users/certik/events{/privacy}",
                "received_events_url": "https://api.github.com/users/certik/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627441,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/Alternate%20Python",
                    "name": "Alternate Python",
                    "color": "800080",
                    "default": false
                },
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 84627389,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/Testing",
                    "name": "Testing",
                    "color": "c7def8",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 39,
            "created_at": "2008-09-21T01:05:27Z",
            "updated_at": "2015-04-17T16:59:21Z",
            "closed_at": null,
            "body": "bc.. The ultimate solution is to provide test access to all developers on our\nbuildbots (the buildbots allow to test only a patch that you send to them)\n& setup parallel testing & provide several levels of testing (core, basic,\nthorough). \n\nAny volunteers to setup buildbots? I'll provide you with necessary access.\n\nI can do that after I finish my finals. http://buildbot.sympy.org/waterfall\n\np. Original issue for \"#4216\":https://github.com/sympy/sympy/issues/4216: \"http://code.google.com/p/sympy/issues/detail?id=1117\":http://code.google.com/p/sympy/issues/detail?id=1117\n\np. Original author: \"https://code.google.com/u/104039945248245758823/\":https://code.google.com/u/104039945248245758823/\n",
            "score": 0.124234945
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/4331",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/4331/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/4331/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/4331/events",
            "html_url": "https://github.com/sympy/sympy/issues/4331",
            "id": 28949579,
            "number": 4331,
            "title": "add jython howto to our docs",
            "user": {
                "login": "certik",
                "id": 20568,
                "avatar_url": "https://avatars0.githubusercontent.com/u/20568?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/certik",
                "html_url": "https://github.com/certik",
                "followers_url": "https://api.github.com/users/certik/followers",
                "following_url": "https://api.github.com/users/certik/following{/other_user}",
                "gists_url": "https://api.github.com/users/certik/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/certik/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/certik/subscriptions",
                "organizations_url": "https://api.github.com/users/certik/orgs",
                "repos_url": "https://api.github.com/users/certik/repos",
                "events_url": "https://api.github.com/users/certik/events{/privacy}",
                "received_events_url": "https://api.github.com/users/certik/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627441,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/Alternate%20Python",
                    "name": "Alternate Python",
                    "color": "800080",
                    "default": false
                },
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 75782144,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/Easy%20to%20Fix",
                    "name": "Easy to Fix",
                    "color": "009800",
                    "default": false
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 6,
            "created_at": "2008-12-06T01:02:51Z",
            "updated_at": "2015-04-17T16:58:12Z",
            "closed_at": null,
            "body": "bc.. From our feedback:\n\nOK.  Thanks.  I finally got it by using jython 2.50b rather\nthan the production version of jython and then running\nthe Windows installer of sympy and issuing 'jython setup.py'\nwhere setup.py is in sympy.  Suggest that an actual console\nsession log showing the installation of jython and sympy be\nincluded in the installation documentation to clarify\nthe steps, which versions of each component are required,\netc.\n\np. Original issue for \"#4331\":https://github.com/sympy/sympy/issues/4331: \"http://code.google.com/p/sympy/issues/detail?id=1232\":http://code.google.com/p/sympy/issues/detail?id=1232\n\np. Original author: \"https://code.google.com/u/104039945248245758823/\":https://code.google.com/u/104039945248245758823/\n",
            "score": 0.33729702
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/4446",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/4446/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/4446/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/4446/events",
            "html_url": "https://github.com/sympy/sympy/issues/4446",
            "id": 28949709,
            "number": 4446,
            "title": "creating lambda evaluator failed",
            "user": {
                "login": "sympy-issue-migrator",
                "id": 6188475,
                "avatar_url": "https://avatars3.githubusercontent.com/u/6188475?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/sympy-issue-migrator",
                "html_url": "https://github.com/sympy-issue-migrator",
                "followers_url": "https://api.github.com/users/sympy-issue-migrator/followers",
                "following_url": "https://api.github.com/users/sympy-issue-migrator/following{/other_user}",
                "gists_url": "https://api.github.com/users/sympy-issue-migrator/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/sympy-issue-migrator/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/sympy-issue-migrator/subscriptions",
                "organizations_url": "https://api.github.com/users/sympy-issue-migrator/orgs",
                "repos_url": "https://api.github.com/users/sympy-issue-migrator/repos",
                "events_url": "https://api.github.com/users/sympy-issue-migrator/events{/privacy}",
                "received_events_url": "https://api.github.com/users/sympy-issue-migrator/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 4,
            "created_at": "2009-03-25T20:57:24Z",
            "updated_at": "2015-04-17T16:58:12Z",
            "closed_at": null,
            "body": "bc.. V 6.5\nthe following code Crash:\n- cannot do a correct loop to display 1000 circle : why catch an exception ?\n  -the labels of the second plot \"p2\" are not correcly displayed \n\nWarning: creating lambda evaluator failed. Falling back on sympy subs\nevaluator.\nWarning: creating lambda evaluator failed. Falling back on sympy subs\nevaluator.\nUncaught exception in event loop: valeur non valide\n\nfrom sympy.geometry.point import Point\nfrom sympy.geometry.ellipse import Circle\nfrom sympy import *\n\np = Plot(axes='label_axes=True', visible=False)\n\nfor i in range (10):\n  try:\n      c=  Circle(Point(0,0), i)\n      p[i] = c\n  except Exception, err:\n      print str(err)\n\np.show()\n\np2 = Plot(axes='label_axes=True')\n\nc = Circle(Point(0,0), 1)\nt = c.tangent_line(c.random_point())\np2[0] = c\np2[1] = t\n\np. Original issue for \"#4446\":https://github.com/sympy/sympy/issues/4446: \"http://code.google.com/p/sympy/issues/detail?id=1347\":http://code.google.com/p/sympy/issues/detail?id=1347\n\np. Original author: \"https://code.google.com/u/102679851029648305776/\":https://code.google.com/u/102679851029648305776/\n",
            "score": 0.40574965
        },
        {
            "url": "https://api.github.com/repos/imrehg/minibook/issues/2",
            "repository_url": "https://api.github.com/repos/imrehg/minibook",
            "labels_url": "https://api.github.com/repos/imrehg/minibook/issues/2/labels{/name}",
            "comments_url": "https://api.github.com/repos/imrehg/minibook/issues/2/comments",
            "events_url": "https://api.github.com/repos/imrehg/minibook/issues/2/events",
            "html_url": "https://github.com/imrehg/minibook/issues/2",
            "id": 15709,
            "number": 2,
            "title": "Icon in app name",
            "user": {
                "login": "imrehg",
                "id": 38863,
                "avatar_url": "https://avatars3.githubusercontent.com/u/38863?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/imrehg",
                "html_url": "https://github.com/imrehg",
                "followers_url": "https://api.github.com/users/imrehg/followers",
                "following_url": "https://api.github.com/users/imrehg/following{/other_user}",
                "gists_url": "https://api.github.com/users/imrehg/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/imrehg/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/imrehg/subscriptions",
                "organizations_url": "https://api.github.com/users/imrehg/orgs",
                "repos_url": "https://api.github.com/users/imrehg/repos",
                "events_url": "https://api.github.com/users/imrehg/events{/privacy}",
                "received_events_url": "https://api.github.com/users/imrehg/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 4944,
                    "url": "https://api.github.com/repos/imrehg/minibook/labels/Bug",
                    "name": "Bug",
                    "color": "e81010",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 1,
            "created_at": "2009-05-16T01:21:33Z",
            "updated_at": "2009-06-07T05:32:03Z",
            "closed_at": null,
            "body": "Currently icon only resides in status bar, but no icon for the task bar part.\n",
            "score": 0.71719646
        },
        {
            "url": "https://api.github.com/repos/imrehg/minibook/issues/7",
            "repository_url": "https://api.github.com/repos/imrehg/minibook",
            "labels_url": "https://api.github.com/repos/imrehg/minibook/issues/7/labels{/name}",
            "comments_url": "https://api.github.com/repos/imrehg/minibook/issues/7/comments",
            "events_url": "https://api.github.com/repos/imrehg/minibook/issues/7/events",
            "html_url": "https://github.com/imrehg/minibook/issues/7",
            "id": 16922,
            "number": 7,
            "title": "Systray icon remains after application close",
            "user": {
                "login": "imrehg",
                "id": 38863,
                "avatar_url": "https://avatars3.githubusercontent.com/u/38863?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/imrehg",
                "html_url": "https://github.com/imrehg",
                "followers_url": "https://api.github.com/users/imrehg/followers",
                "following_url": "https://api.github.com/users/imrehg/following{/other_user}",
                "gists_url": "https://api.github.com/users/imrehg/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/imrehg/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/imrehg/subscriptions",
                "organizations_url": "https://api.github.com/users/imrehg/orgs",
                "repos_url": "https://api.github.com/users/imrehg/repos",
                "events_url": "https://api.github.com/users/imrehg/events{/privacy}",
                "received_events_url": "https://api.github.com/users/imrehg/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 4944,
                    "url": "https://api.github.com/repos/imrehg/minibook/labels/Bug",
                    "name": "Bug",
                    "color": "e81010",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 0,
            "created_at": "2009-05-19T15:50:52Z",
            "updated_at": "2009-06-07T05:32:03Z",
            "closed_at": null,
            "body": "Systray icon remains after the application is closed. Mousover makes it disappear.\nNoticed on windows, not sure if it's the same under Linux as well...\n",
            "score": 0.8229122
        },
        {
            "url": "https://api.github.com/repos/imrehg/mitter-fork/issues/2",
            "repository_url": "https://api.github.com/repos/imrehg/mitter-fork",
            "labels_url": "https://api.github.com/repos/imrehg/mitter-fork/issues/2/labels{/name}",
            "comments_url": "https://api.github.com/repos/imrehg/mitter-fork/issues/2/comments",
            "events_url": "https://api.github.com/repos/imrehg/mitter-fork/issues/2/events",
            "html_url": "https://github.com/imrehg/mitter-fork/issues/2",
            "id": 19353,
            "number": 2,
            "title": "Threading fix for windows",
            "user": {
                "login": "imrehg",
                "id": 38863,
                "avatar_url": "https://avatars3.githubusercontent.com/u/38863?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/imrehg",
                "html_url": "https://github.com/imrehg",
                "followers_url": "https://api.github.com/users/imrehg/followers",
                "following_url": "https://api.github.com/users/imrehg/following{/other_user}",
                "gists_url": "https://api.github.com/users/imrehg/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/imrehg/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/imrehg/subscriptions",
                "organizations_url": "https://api.github.com/users/imrehg/orgs",
                "repos_url": "https://api.github.com/users/imrehg/repos",
                "events_url": "https://api.github.com/users/imrehg/events{/privacy}",
                "received_events_url": "https://api.github.com/users/imrehg/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 5812,
                    "url": "https://api.github.com/repos/imrehg/mitter-fork/labels/Bug",
                    "name": "Bug",
                    "color": "f70000",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 0,
            "created_at": "2009-05-26T04:40:00Z",
            "updated_at": "2009-05-26T04:40:00Z",
            "closed_at": null,
            "body": "Check proper threading enter() and leave() for the code, not to have the entire program inside that...\n",
            "score": 6.2315407
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/4563",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/4563/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/4563/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/4563/events",
            "html_url": "https://github.com/sympy/sympy/issues/4563",
            "id": 28949846,
            "number": 4563,
            "title": "docs: adding instructions how to get stuff working on windows",
            "user": {
                "login": "certik",
                "id": 20568,
                "avatar_url": "https://avatars0.githubusercontent.com/u/20568?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/certik",
                "html_url": "https://github.com/certik",
                "followers_url": "https://api.github.com/users/certik/followers",
                "following_url": "https://api.github.com/users/certik/following{/other_user}",
                "gists_url": "https://api.github.com/users/certik/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/certik/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/certik/subscriptions",
                "organizations_url": "https://api.github.com/users/certik/orgs",
                "repos_url": "https://api.github.com/users/certik/repos",
                "events_url": "https://api.github.com/users/certik/events{/privacy}",
                "received_events_url": "https://api.github.com/users/certik/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 75863299,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/Documentation",
                    "name": "Documentation",
                    "color": "c7def8",
                    "default": false
                },
                {
                    "id": 75782144,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/Easy%20to%20Fix",
                    "name": "Easy to Fix",
                    "color": "009800",
                    "default": false
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 7,
            "created_at": "2009-06-09T06:06:53Z",
            "updated_at": "2016-02-25T19:02:17Z",
            "closed_at": null,
            "body": "```\nSee this thread for more info: http://groups.google.com/group/sympy/browse_thread/thread/de26d886c983fbec\n```\n\nOriginal issue for #4563: http://code.google.com/p/sympy/issues/detail?id=1464\nOriginal author: https://code.google.com/u/104039945248245758823/\n",
            "score": 3.9501157
        },
        {
            "url": "https://api.github.com/repos/batterseapower/pinyin-toolkit/issues/132",
            "repository_url": "https://api.github.com/repos/batterseapower/pinyin-toolkit",
            "labels_url": "https://api.github.com/repos/batterseapower/pinyin-toolkit/issues/132/labels{/name}",
            "comments_url": "https://api.github.com/repos/batterseapower/pinyin-toolkit/issues/132/comments",
            "events_url": "https://api.github.com/repos/batterseapower/pinyin-toolkit/issues/132/events",
            "html_url": "https://github.com/batterseapower/pinyin-toolkit/issues/132",
            "id": 35802,
            "number": 132,
            "title": "Line Number Indexes Beyond 20 Not Displayed",
            "user": {
                "login": "Nick3C",
                "id": 90254,
                "avatar_url": "https://avatars0.githubusercontent.com/u/90254?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/Nick3C",
                "html_url": "https://github.com/Nick3C",
                "followers_url": "https://api.github.com/users/Nick3C/followers",
                "following_url": "https://api.github.com/users/Nick3C/following{/other_user}",
                "gists_url": "https://api.github.com/users/Nick3C/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/Nick3C/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/Nick3C/subscriptions",
                "organizations_url": "https://api.github.com/users/Nick3C/orgs",
                "repos_url": "https://api.github.com/users/Nick3C/repos",
                "events_url": "https://api.github.com/users/Nick3C/events{/privacy}",
                "received_events_url": "https://api.github.com/users/Nick3C/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 6425,
                    "url": "https://api.github.com/repos/batterseapower/pinyin-toolkit/labels/bug",
                    "name": "bug",
                    "color": "ff0000",
                    "default": true
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 15,
            "created_at": "2009-07-12T20:10:41Z",
            "updated_at": "2009-07-19T09:23:43Z",
            "closed_at": null,
            "body": "I am not totally sure about the new line numbers. What font are you using to display them? Arial Unicode doesn't support them (and it's the most comprehensive windows font I know of).\n\nIf we can find a commonly distributed windows font that supports them then no problem (we can use html font tags) but otherwise the \"(21)\" style is probably better.\n",
            "score": 0.75318795
        },
        {
            "url": "https://api.github.com/repos/gak/pycallgraph/issues/84",
            "repository_url": "https://api.github.com/repos/gak/pycallgraph",
            "labels_url": "https://api.github.com/repos/gak/pycallgraph/issues/84/labels{/name}",
            "comments_url": "https://api.github.com/repos/gak/pycallgraph/issues/84/comments",
            "events_url": "https://api.github.com/repos/gak/pycallgraph/issues/84/events",
            "html_url": "https://github.com/gak/pycallgraph/issues/84",
            "id": 46451,
            "number": 84,
            "title": "Escape pathnames for dot",
            "user": {
                "login": "gak",
                "id": 31338,
                "avatar_url": "https://avatars0.githubusercontent.com/u/31338?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/gak",
                "html_url": "https://github.com/gak",
                "followers_url": "https://api.github.com/users/gak/followers",
                "following_url": "https://api.github.com/users/gak/following{/other_user}",
                "gists_url": "https://api.github.com/users/gak/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/gak/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/gak/subscriptions",
                "organizations_url": "https://api.github.com/users/gak/orgs",
                "repos_url": "https://api.github.com/users/gak/repos",
                "events_url": "https://api.github.com/users/gak/events{/privacy}",
                "received_events_url": "https://api.github.com/users/gak/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 54208387,
                    "url": "https://api.github.com/repos/gak/pycallgraph/labels/bug",
                    "name": "bug",
                    "color": "5319e7",
                    "default": true
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": {
                "url": "https://api.github.com/repos/gak/pycallgraph/milestones/2",
                "html_url": "https://github.com/gak/pycallgraph/milestone/2",
                "labels_url": "https://api.github.com/repos/gak/pycallgraph/milestones/2/labels",
                "id": 385685,
                "number": 2,
                "title": "1.1.0",
                "description": "Realtime callgraphs",
                "creator": {
                    "login": "gak",
                    "id": 31338,
                    "avatar_url": "https://avatars0.githubusercontent.com/u/31338?v=3",
                    "gravatar_id": "",
                    "url": "https://api.github.com/users/gak",
                    "html_url": "https://github.com/gak",
                    "followers_url": "https://api.github.com/users/gak/followers",
                    "following_url": "https://api.github.com/users/gak/following{/other_user}",
                    "gists_url": "https://api.github.com/users/gak/gists{/gist_id}",
                    "starred_url": "https://api.github.com/users/gak/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/gak/subscriptions",
                    "organizations_url": "https://api.github.com/users/gak/orgs",
                    "repos_url": "https://api.github.com/users/gak/repos",
                    "events_url": "https://api.github.com/users/gak/events{/privacy}",
                    "received_events_url": "https://api.github.com/users/gak/received_events",
                    "type": "User",
                    "site_admin": false
                },
                "open_issues": 10,
                "closed_issues": 0,
                "state": "open",
                "created_at": "2013-07-24T08:58:17Z",
                "updated_at": "2013-12-13T21:16:10Z",
                "due_on": "2014-02-28T08:00:00Z",
                "closed_at": null
            },
            "comments": 0,
            "created_at": "2009-08-12T12:20:44Z",
            "updated_at": "2013-09-14T06:35:51Z",
            "closed_at": null,
            "body": "In my first test pycallpraph barfed spectacularily (on windows XP/german), because it just fed the pathname for the picture - containing spaces - to dot. The easiest way to fix that is using the subprocess module (in the standard library since python 2.4).\n\n(My version is actually 0.5.1 installed via easy_install but I can't select it under \"Version\" for this ticket)\n",
            "score": 0.46282244
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/4724",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/4724/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/4724/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/4724/events",
            "html_url": "https://github.com/sympy/sympy/issues/4724",
            "id": 28950057,
            "number": 4724,
            "title": "sympify of logical operations gets confused with ==",
            "user": {
                "login": "sympy-issue-migrator",
                "id": 6188475,
                "avatar_url": "https://avatars3.githubusercontent.com/u/6188475?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/sympy-issue-migrator",
                "html_url": "https://github.com/sympy-issue-migrator",
                "followers_url": "https://api.github.com/users/sympy-issue-migrator/followers",
                "following_url": "https://api.github.com/users/sympy-issue-migrator/following{/other_user}",
                "gists_url": "https://api.github.com/users/sympy-issue-migrator/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/sympy-issue-migrator/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/sympy-issue-migrator/subscriptions",
                "organizations_url": "https://api.github.com/users/sympy-issue-migrator/orgs",
                "repos_url": "https://api.github.com/users/sympy-issue-migrator/repos",
                "events_url": "https://api.github.com/users/sympy-issue-migrator/events{/privacy}",
                "received_events_url": "https://api.github.com/users/sympy-issue-migrator/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 191924,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/Needs%20Better%20Patch",
                    "name": "Needs Better Patch",
                    "color": "fbca04",
                    "default": false
                },
                {
                    "id": 84627362,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/parsing",
                    "name": "parsing",
                    "color": "0000FF",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 10,
            "created_at": "2009-09-01T04:11:45Z",
            "updated_at": "2015-04-17T16:58:12Z",
            "closed_at": null,
            "body": "```\nI am using sympy 0.6.5 on linux but I see the behavior on windows too.\n\nIn [10]: sympy.sympify('A>2')\nOut[10]: 2 < A\n\nIn [11]: sympy.sympify('A==2')\nOut[11]: False\n\n\nI even tried using the verbose \"Equality\" function:\n\nThis works and returns an expression that prints a \"A == 2\"\nIn [16]: sympy.sympify('Equality(A,2)')\nOut[16]: A == 2\n\nBut \"A == 2\" will not sympify\nIn [17]: sympy.sympify('A == 2')\nOut[17]: False\n\nUsing Equality() is very verbose so I was hoping that it was possible to \nget == working.\n\n--Tom\n```\n\nOriginal issue for #4724: http://code.google.com/p/sympy/issues/detail?id=1625\nOriginal author: https://code.google.com/u/110078974596186779257/\n",
            "score": 0.33729702
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/4748",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/4748/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/4748/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/4748/events",
            "html_url": "https://github.com/sympy/sympy/issues/4748",
            "id": 28950088,
            "number": 4748,
            "title": "Intersect bug (feature)",
            "user": {
                "login": "hut3",
                "id": 134333,
                "avatar_url": "https://avatars2.githubusercontent.com/u/134333?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/hut3",
                "html_url": "https://github.com/hut3",
                "followers_url": "https://api.github.com/users/hut3/followers",
                "following_url": "https://api.github.com/users/hut3/following{/other_user}",
                "gists_url": "https://api.github.com/users/hut3/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/hut3/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/hut3/subscriptions",
                "organizations_url": "https://api.github.com/users/hut3/orgs",
                "repos_url": "https://api.github.com/users/hut3/repos",
                "events_url": "https://api.github.com/users/hut3/events{/privacy}",
                "received_events_url": "https://api.github.com/users/hut3/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 4,
            "created_at": "2009-09-17T11:53:10Z",
            "updated_at": "2015-04-17T16:58:12Z",
            "closed_at": null,
            "body": "```\nIntersect seems to generally work incorrectly, example is:\n\nprint intersection(Segment(Point(5, -0.730642868656981), Point(6,\n-0.0988049466141603)), Segment(Point(5.5, -2.74551357555141), Point(5.5, 1)))\n\nSo the intersection should be at about (5.5, -1.4)?\n\nSeems to be broken in general, works rarely.  I can give more example code.\n\nSorry if I'm overtired right now and doing something blatantly stupid.\n```\n\nOriginal issue for #4748: http://code.google.com/p/sympy/issues/detail?id=1649\nOriginal author: https://code.google.com/u/106893876112355965992/\n",
            "score": 0.1793177
        },
        {
            "url": "https://api.github.com/repos/nvaccess/nvda/issues/419",
            "repository_url": "https://api.github.com/repos/nvaccess/nvda",
            "labels_url": "https://api.github.com/repos/nvaccess/nvda/issues/419/labels{/name}",
            "comments_url": "https://api.github.com/repos/nvaccess/nvda/issues/419/comments",
            "events_url": "https://api.github.com/repos/nvaccess/nvda/issues/419/events",
            "html_url": "https://github.com/nvaccess/nvda/issues/419",
            "id": 116022216,
            "number": 419,
            "title": "Unable to read text in hidden textbox",
            "user": {
                "login": "nvaccessAuto",
                "id": 14253915,
                "avatar_url": "https://avatars1.githubusercontent.com/u/14253915?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/nvaccessAuto",
                "html_url": "https://github.com/nvaccessAuto",
                "followers_url": "https://api.github.com/users/nvaccessAuto/followers",
                "following_url": "https://api.github.com/users/nvaccessAuto/following{/other_user}",
                "gists_url": "https://api.github.com/users/nvaccessAuto/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/nvaccessAuto/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/nvaccessAuto/subscriptions",
                "organizations_url": "https://api.github.com/users/nvaccessAuto/orgs",
                "repos_url": "https://api.github.com/users/nvaccessAuto/repos",
                "events_url": "https://api.github.com/users/nvaccessAuto/events{/privacy}",
                "received_events_url": "https://api.github.com/users/nvaccessAuto/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 284598354,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/App%20specific:%20Microsoft%20Visual%20Studio",
                    "name": "App specific: Microsoft Visual Studio",
                    "color": "ffffff",
                    "default": false
                },
                {
                    "id": 284590813,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/bug",
                    "name": "bug",
                    "color": "fc2929",
                    "default": true
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 0,
            "created_at": "2010-01-01T00:00:00Z",
            "updated_at": "2015-11-10T04:14:59Z",
            "closed_at": null,
            "body": "**Reported by aarnott on 2009-09-18 22:35**\nA hidden text control is embedded on the Visual Studio (2008 and 2010 beta 1) Project Conversion dialog (opening page).  Windows Eyes can read it, but NVDA cannot.\n",
            "score": 0.6790957
        },
        {
            "url": "https://api.github.com/repos/nvaccess/nvda/issues/513",
            "repository_url": "https://api.github.com/repos/nvaccess/nvda",
            "labels_url": "https://api.github.com/repos/nvaccess/nvda/issues/513/labels{/name}",
            "comments_url": "https://api.github.com/repos/nvaccess/nvda/issues/513/comments",
            "events_url": "https://api.github.com/repos/nvaccess/nvda/issues/513/events",
            "html_url": "https://github.com/nvaccess/nvda/issues/513",
            "id": 116022543,
            "number": 513,
            "title": "Some output in command consoels spelled with speak typed characters enabled",
            "user": {
                "login": "nvaccessAuto",
                "id": 14253915,
                "avatar_url": "https://avatars1.githubusercontent.com/u/14253915?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/nvaccessAuto",
                "html_url": "https://github.com/nvaccessAuto",
                "followers_url": "https://api.github.com/users/nvaccessAuto/followers",
                "following_url": "https://api.github.com/users/nvaccessAuto/following{/other_user}",
                "gists_url": "https://api.github.com/users/nvaccessAuto/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/nvaccessAuto/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/nvaccessAuto/subscriptions",
                "organizations_url": "https://api.github.com/users/nvaccessAuto/orgs",
                "repos_url": "https://api.github.com/users/nvaccessAuto/repos",
                "events_url": "https://api.github.com/users/nvaccessAuto/events{/privacy}",
                "received_events_url": "https://api.github.com/users/nvaccessAuto/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 284590813,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/bug",
                    "name": "bug",
                    "color": "fc2929",
                    "default": true
                },
                {
                    "id": 401238340,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/p2",
                    "name": "p2",
                    "color": "5319e7",
                    "default": false
                },
                {
                    "id": 284596914,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/Windows%20command%20console",
                    "name": "Windows command console",
                    "color": "ffffff",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 12,
            "created_at": "2010-01-01T00:00:00Z",
            "updated_at": "2016-07-06T00:25:51Z",
            "closed_at": null,
            "body": "**Reported by aleksey_s on 2009-12-17 16:39**\nSometimes, NVDA is too verbose in console windows.\nSTR:\n- open cmd\n- run python interpreter\n- type: print \"hi\"\n  Currently, NVDA says\n  h\n  i\n  hi\n  It starts reading by letter and only then reads the entire message. This behaviour also exists in other console apps.\n  **Blocked by #5027**\n  **Blocking #2977, #4127, #5328**\n",
            "score": 0.5586072
        },
        {
            "url": "https://api.github.com/repos/nvaccess/nvda/issues/446",
            "repository_url": "https://api.github.com/repos/nvaccess/nvda",
            "labels_url": "https://api.github.com/repos/nvaccess/nvda/issues/446/labels{/name}",
            "comments_url": "https://api.github.com/repos/nvaccess/nvda/issues/446/comments",
            "events_url": "https://api.github.com/repos/nvaccess/nvda/issues/446/events",
            "html_url": "https://github.com/nvaccess/nvda/issues/446",
            "id": 116022302,
            "number": 446,
            "title": "When deleting one before the last message in the outlook express message list NVDA fails to announce new item in focus",
            "user": {
                "login": "nvaccessAuto",
                "id": 14253915,
                "avatar_url": "https://avatars1.githubusercontent.com/u/14253915?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/nvaccessAuto",
                "html_url": "https://github.com/nvaccessAuto",
                "followers_url": "https://api.github.com/users/nvaccessAuto/followers",
                "following_url": "https://api.github.com/users/nvaccessAuto/following{/other_user}",
                "gists_url": "https://api.github.com/users/nvaccessAuto/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/nvaccessAuto/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/nvaccessAuto/subscriptions",
                "organizations_url": "https://api.github.com/users/nvaccessAuto/orgs",
                "repos_url": "https://api.github.com/users/nvaccessAuto/repos",
                "events_url": "https://api.github.com/users/nvaccessAuto/events{/privacy}",
                "received_events_url": "https://api.github.com/users/nvaccessAuto/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 284590813,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/bug",
                    "name": "bug",
                    "color": "fc2929",
                    "default": true
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 2,
            "created_at": "2010-01-01T00:00:00Z",
            "updated_at": "2015-11-10T04:16:01Z",
            "closed_at": null,
            "body": "**Reported by pvagner on 2009-10-29 19:14**\nSTR:\n1) Open your outlook express and focus the list of messages.\n2) Make sure focus is set to the item immediately before the last item in the list (the second item from the very bottom).\n3) Press the delete key.\n\nActual results:\nNVDA stays silent and you need to use NVDA+tab or NVDA+up arrow to get the currently selected item spoken.\n\nExpected results:\nAs the last message in the list gains the focus, NVDA should correctly announce to that fact.\n",
            "score": 0.5102169
        },
        {
            "url": "https://api.github.com/repos/nvaccess/nvda/issues/510",
            "repository_url": "https://api.github.com/repos/nvaccess/nvda",
            "labels_url": "https://api.github.com/repos/nvaccess/nvda/issues/510/labels{/name}",
            "comments_url": "https://api.github.com/repos/nvaccess/nvda/issues/510/comments",
            "events_url": "https://api.github.com/repos/nvaccess/nvda/issues/510/events",
            "html_url": "https://github.com/nvaccess/nvda/issues/510",
            "id": 116022534,
            "number": 510,
            "title": "Unclear indication of text deletion",
            "user": {
                "login": "nvaccessAuto",
                "id": 14253915,
                "avatar_url": "https://avatars1.githubusercontent.com/u/14253915?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/nvaccessAuto",
                "html_url": "https://github.com/nvaccessAuto",
                "followers_url": "https://api.github.com/users/nvaccessAuto/followers",
                "following_url": "https://api.github.com/users/nvaccessAuto/following{/other_user}",
                "gists_url": "https://api.github.com/users/nvaccessAuto/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/nvaccessAuto/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/nvaccessAuto/subscriptions",
                "organizations_url": "https://api.github.com/users/nvaccessAuto/orgs",
                "repos_url": "https://api.github.com/users/nvaccessAuto/repos",
                "events_url": "https://api.github.com/users/nvaccessAuto/events{/privacy}",
                "received_events_url": "https://api.github.com/users/nvaccessAuto/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 284590813,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/bug",
                    "name": "bug",
                    "color": "fc2929",
                    "default": true
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 4,
            "created_at": "2010-01-01T00:00:00Z",
            "updated_at": "2015-11-10T04:18:22Z",
            "closed_at": null,
            "body": "**Reported by Hari on 2009-12-15 15:11**\nWhen deleting a selected text in any word pad/text editor, the NVDA, version 2009.1 (most recent release), says: \"unselecting\" followed by announcement of a word if present -- and if not, announcement of a carriage return. E.g., let's say we have following words written: \"my test\". And let's say we deleted the word \"my\", the NVDA says \"unselecting test\", which, you could see, is quite confusing. Now let's say the text is followed by a carriage return, it say the following: \"unselecting carriage return\". As noted before, this issue is global, i.e. it occurs in MS Word, Windows Word Pad, Jarte text editor, and so forth. \n",
            "score": 0.3571519
        },
        {
            "url": "https://api.github.com/repos/nvaccess/nvda/issues/400",
            "repository_url": "https://api.github.com/repos/nvaccess/nvda",
            "labels_url": "https://api.github.com/repos/nvaccess/nvda/issues/400/labels{/name}",
            "comments_url": "https://api.github.com/repos/nvaccess/nvda/issues/400/comments",
            "events_url": "https://api.github.com/repos/nvaccess/nvda/issues/400/events",
            "html_url": "https://github.com/nvaccess/nvda/issues/400",
            "id": 116022151,
            "number": 400,
            "title": "NVDA prevents automatic system standby",
            "user": {
                "login": "nvaccessAuto",
                "id": 14253915,
                "avatar_url": "https://avatars1.githubusercontent.com/u/14253915?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/nvaccessAuto",
                "html_url": "https://github.com/nvaccessAuto",
                "followers_url": "https://api.github.com/users/nvaccessAuto/followers",
                "following_url": "https://api.github.com/users/nvaccessAuto/following{/other_user}",
                "gists_url": "https://api.github.com/users/nvaccessAuto/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/nvaccessAuto/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/nvaccessAuto/subscriptions",
                "organizations_url": "https://api.github.com/users/nvaccessAuto/orgs",
                "repos_url": "https://api.github.com/users/nvaccessAuto/repos",
                "events_url": "https://api.github.com/users/nvaccessAuto/events{/privacy}",
                "received_events_url": "https://api.github.com/users/nvaccessAuto/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 284590813,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/bug",
                    "name": "bug",
                    "color": "fc2929",
                    "default": true
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 3,
            "created_at": "2010-01-01T00:00:00Z",
            "updated_at": "2015-11-10T04:14:15Z",
            "closed_at": null,
            "body": "**Reported by jteh on 2009-08-25 03:42**\nWhile NVDA is running, the system will not go into standby mode automatically despite being configured to do so.\n\nStr:\n1. Ensure that no other background tasks which will prevent standby are running; e.g. audio players, etc.\n2. Set the system standby time to 1 minute. (In Windows XP, Control Panel -> Power Options -> Power Schemes tab -> System standby...)\n3. Press NVDA+f12 and let the speech reach completion. Do not hit control to interrupt it. Otherwise, the speech will pause instead of finishing.\n4. Wait a little more than 1 minute.\n   Actual: The system does not go into standby mode.\n   Expected: The system should go into standby mode after 1 minute.\n\nAdditional info:\nThis occurs because tones keeps an instance of `nvwave.WavePlayer` which it does not close after each tone plays. We do this so that we don't have to (synchronously) wait for the tone to finish playing. Calling `tones.player.close()` from the Python Console fixes this problem. Fixing this will require refactoring the tones code so that it calls `player.idle()` after each tone, which will probably need to be done in a background thread.\n",
            "score": 0.28862226
        },
        {
            "url": "https://api.github.com/repos/nvaccess/nvda/issues/500",
            "repository_url": "https://api.github.com/repos/nvaccess/nvda",
            "labels_url": "https://api.github.com/repos/nvaccess/nvda/issues/500/labels{/name}",
            "comments_url": "https://api.github.com/repos/nvaccess/nvda/issues/500/comments",
            "events_url": "https://api.github.com/repos/nvaccess/nvda/issues/500/events",
            "html_url": "https://github.com/nvaccess/nvda/issues/500",
            "id": 116022500,
            "number": 500,
            "title": "Cdex is not beeping in background or foreground  progress bars",
            "user": {
                "login": "nvaccessAuto",
                "id": 14253915,
                "avatar_url": "https://avatars1.githubusercontent.com/u/14253915?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/nvaccessAuto",
                "html_url": "https://github.com/nvaccessAuto",
                "followers_url": "https://api.github.com/users/nvaccessAuto/followers",
                "following_url": "https://api.github.com/users/nvaccessAuto/following{/other_user}",
                "gists_url": "https://api.github.com/users/nvaccessAuto/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/nvaccessAuto/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/nvaccessAuto/subscriptions",
                "organizations_url": "https://api.github.com/users/nvaccessAuto/orgs",
                "repos_url": "https://api.github.com/users/nvaccessAuto/repos",
                "events_url": "https://api.github.com/users/nvaccessAuto/events{/privacy}",
                "received_events_url": "https://api.github.com/users/nvaccessAuto/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 284590813,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/bug",
                    "name": "bug",
                    "color": "fc2929",
                    "default": true
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 3,
            "created_at": "2010-01-01T00:00:00Z",
            "updated_at": "2015-11-10T04:18:00Z",
            "closed_at": null,
            "body": "**Reported by scottbb on 2009-12-07 02:46**\nWhen using Cdex there is no beeping for the background progress bar.\n",
            "score": 0.2525445
        },
        {
            "url": "https://api.github.com/repos/cherrypy/cherrypy/issues/983",
            "repository_url": "https://api.github.com/repos/cherrypy/cherrypy",
            "labels_url": "https://api.github.com/repos/cherrypy/cherrypy/issues/983/labels{/name}",
            "comments_url": "https://api.github.com/repos/cherrypy/cherrypy/issues/983/comments",
            "events_url": "https://api.github.com/repos/cherrypy/cherrypy/issues/983/events",
            "html_url": "https://github.com/cherrypy/cherrypy/issues/983",
            "id": 152024778,
            "number": 983,
            "title": "serving non-existent favicon.ico with staticfile throws error on refresh",
            "user": {
                "login": "bb-migration",
                "id": 18138808,
                "avatar_url": "https://avatars1.githubusercontent.com/u/18138808?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/bb-migration",
                "html_url": "https://github.com/bb-migration",
                "followers_url": "https://api.github.com/users/bb-migration/followers",
                "following_url": "https://api.github.com/users/bb-migration/following{/other_user}",
                "gists_url": "https://api.github.com/users/bb-migration/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/bb-migration/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/bb-migration/subscriptions",
                "organizations_url": "https://api.github.com/users/bb-migration/orgs",
                "repos_url": "https://api.github.com/users/bb-migration/repos",
                "events_url": "https://api.github.com/users/bb-migration/events{/privacy}",
                "received_events_url": "https://api.github.com/users/bb-migration/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 366608106,
                    "url": "https://api.github.com/repos/cherrypy/cherrypy/labels/bug",
                    "name": "bug",
                    "color": "ee0701",
                    "default": true
                },
                {
                    "id": 366608297,
                    "url": "https://api.github.com/repos/cherrypy/cherrypy/labels/CherryPy%20code",
                    "name": "CherryPy code",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 366608109,
                    "url": "https://api.github.com/repos/cherrypy/cherrypy/labels/help%20wanted",
                    "name": "help wanted",
                    "color": "128A0C",
                    "default": true
                },
                {
                    "id": 366608295,
                    "url": "https://api.github.com/repos/cherrypy/cherrypy/labels/major",
                    "name": "major",
                    "color": "ededed",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 6,
            "created_at": "2010-01-17T06:47:16Z",
            "updated_at": "2017-02-03T16:56:35Z",
            "closed_at": null,
            "body": "Originally reported by: **@wyuenho**\n\n---\n\nSo apparently Tree.mount() attaches a static handler for favicon.ico if the root doesn't already have a handler for it without even looking at the config. If the config has a section for /favicon.ico, but the file doesn't exist, _refreshing_ the browser will throw and error in the errorlog with something like this:\n\n```\n[17/Jan/2010:01:30:00] HTTP Traceback (most recent call last):\n  File \"/Users/wyuenho/Documents/spiralup/spiralup/lib/python2.5/site-packages/CherryPy-3.1.2-py2.5.egg/cherrypy/_cprequest.py\", line 606, in respond\n    cherrypy.response.body = self.handler()\n  File \"/Users/wyuenho/Documents/spiralup/spiralup/lib/python2.5/site-packages/CherryPy-3.1.2-py2.5.egg/cherrypy/_cpdispatch.py\", line 25, in __call__\n    return self.callable(*self.args, **self.kwargs)\n  File \"/Users/wyuenho/Documents/spiralup/spiralup/lib/python2.5/site-packages/CherryPy-3.1.2-py2.5.egg/cherrypy/_cptools.py\", line 150, in handle_func\n    handled = self.callable(*args, **self._merged_args(kwargs))\nTypeError: staticfile() got multiple values for keyword argument 'filename'\n```\n\nThis obviously doesn't even tell me what values it got, so the error message doesn't even help. I'm not sure how to patch this because you need to obviously look at all the different kinds of dispatchers for the \"/favicon.ico\" key besides the default tree dispatcher, and I'm not familiar with all of them.\n\nReported by wyuenho@gmail.com\n\n---\n- Bitbucket: https://bitbucket.org/cherrypy/cherrypy/issue/983\n",
            "score": 0.29482016
        },
        {
            "url": "https://api.github.com/repos/nvaccess/nvda/issues/533",
            "repository_url": "https://api.github.com/repos/nvaccess/nvda",
            "labels_url": "https://api.github.com/repos/nvaccess/nvda/issues/533/labels{/name}",
            "comments_url": "https://api.github.com/repos/nvaccess/nvda/issues/533/comments",
            "events_url": "https://api.github.com/repos/nvaccess/nvda/issues/533/events",
            "html_url": "https://github.com/nvaccess/nvda/issues/533",
            "id": 116022614,
            "number": 533,
            "title": "NVDA is freezing while activating The dialog window which is awailable in The ffdshow audio decoder. This issue can cause that Nvda can not be used. Or NVDA can respond after 8 or more seconds. The algorithms, which are responsible for getting The text from The dialog Window are very complex and are allocating many system resources until The text from this dialog box is ready for speech synthesizer or Brltty.",
            "user": {
                "login": "nvaccessAuto",
                "id": 14253915,
                "avatar_url": "https://avatars1.githubusercontent.com/u/14253915?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/nvaccessAuto",
                "html_url": "https://github.com/nvaccessAuto",
                "followers_url": "https://api.github.com/users/nvaccessAuto/followers",
                "following_url": "https://api.github.com/users/nvaccessAuto/following{/other_user}",
                "gists_url": "https://api.github.com/users/nvaccessAuto/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/nvaccessAuto/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/nvaccessAuto/subscriptions",
                "organizations_url": "https://api.github.com/users/nvaccessAuto/orgs",
                "repos_url": "https://api.github.com/users/nvaccessAuto/repos",
                "events_url": "https://api.github.com/users/nvaccessAuto/events{/privacy}",
                "received_events_url": "https://api.github.com/users/nvaccessAuto/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 284590813,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/bug",
                    "name": "bug",
                    "color": "fc2929",
                    "default": true
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 1,
            "created_at": "2010-01-18T20:45:39Z",
            "updated_at": "2015-11-10T04:19:17Z",
            "closed_at": null,
            "body": "**Reported by janusz on 2010-01-18 20:45**\nThe algorithms, which are responsible for getting text information from The dialog Window, which can be accessed by recalling ffdshow  audio decoder. When user is triing to activate this dialog window, NVDA can freeze or user must wait more than 7 seconds until NVDA is ready for interactioning with user. This issue is also presented, if user is triing to use INSERT and 4 or 6 key, which is awailable on The numeric key pad. The Klitecodecpack, where The ffdshow filter is awailable can be downloaded by using The following direct link. \n\nftp.tw.xemacs.org/cpatch/media/k-lite/klcodec561f.exe\n\nThe package download syze is 14,7 MB. It is necessary to install this package, please let installer to chose The default settings. To spend Yours work and to free Your time from downloading and using compatible multimedia player, You can activate The dialog Window by exactly running The following command. I love procedures direct calls from dynamic linked libraries by using rundll32.exe command. It is a little pleasure for programmers.\n\nC:\\WINDOWS\\system32\\rundll32.exe ffdshow.ax,configureAudio\n\nThis command will not run if You will not execute it from The  \n\n\\Program Files\\K-Lite Codec Pack\\ffdshow\n\nThe installation program for Klite codecpack will create necessary links for activating some configuration options without need to run compatible multimedia player. Please explore The Programs folder detaily and You will find out, which special links have been created. Please, try to fix this issue. Many visually impaired users would be very deeply appreciating The programmer, who would create new application setting file specifically for all dialog windows, which are awailable for managing all codecs and filters. The purpose of this new application settings file is to force NVDA to get The labels of all editable fields and other important user interface controls. Because The source code of those dlls is frelly awailable, it is very nice C programming language complex source code, somebody would get those labels by analysing all source code or somebody would try to get in touch with The core developers of The Klitecodecpacks and all dlls. This codecs is ammazing, visually impaired user can modifi The sound while plaing multimedia content without need to load The multimedia file to The sound editor. The dolby filter and other modules are working reliably. Be very careful while exploring and experimenting with all numbers for modifiing The equalizer. IN some relationships between various settings, Your computer would freeze and You would have to be forced to turned it of force. \n",
            "score": 0.28811592
        },
        {
            "url": "https://api.github.com/repos/nvaccess/nvda/issues/556",
            "repository_url": "https://api.github.com/repos/nvaccess/nvda",
            "labels_url": "https://api.github.com/repos/nvaccess/nvda/issues/556/labels{/name}",
            "comments_url": "https://api.github.com/repos/nvaccess/nvda/issues/556/comments",
            "events_url": "https://api.github.com/repos/nvaccess/nvda/issues/556/events",
            "html_url": "https://github.com/nvaccess/nvda/issues/556",
            "id": 116022712,
            "number": 556,
            "title": "Keyboard layout change is not reported in console windows",
            "user": {
                "login": "nvaccessAuto",
                "id": 14253915,
                "avatar_url": "https://avatars1.githubusercontent.com/u/14253915?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/nvaccessAuto",
                "html_url": "https://github.com/nvaccessAuto",
                "followers_url": "https://api.github.com/users/nvaccessAuto/followers",
                "following_url": "https://api.github.com/users/nvaccessAuto/following{/other_user}",
                "gists_url": "https://api.github.com/users/nvaccessAuto/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/nvaccessAuto/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/nvaccessAuto/subscriptions",
                "organizations_url": "https://api.github.com/users/nvaccessAuto/orgs",
                "repos_url": "https://api.github.com/users/nvaccessAuto/repos",
                "events_url": "https://api.github.com/users/nvaccessAuto/events{/privacy}",
                "received_events_url": "https://api.github.com/users/nvaccessAuto/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 284590813,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/bug",
                    "name": "bug",
                    "color": "fc2929",
                    "default": true
                },
                {
                    "id": 284596914,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/Windows%20command%20console",
                    "name": "Windows command console",
                    "color": "ffffff",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 2,
            "created_at": "2010-02-13T08:34:34Z",
            "updated_at": "2015-11-10T04:20:11Z",
            "closed_at": null,
            "body": "**Reported by aleksey_s on 2010-02-13 08:34**\n## steps to reproduce\n- ensure you have more than one keyboard layout installed\n- start->run->cmd\n- type something and try to change layout\n- ensure you are entering letters in newly set language\n  ## current results\n\nNVDA doesn't report the change to keyboard layout\n## expected results\n\nNVDA must gracefully report the change\n",
            "score": 3.9446173
        },
        {
            "url": "https://api.github.com/repos/nvaccess/nvda/issues/562",
            "repository_url": "https://api.github.com/repos/nvaccess/nvda",
            "labels_url": "https://api.github.com/repos/nvaccess/nvda/issues/562/labels{/name}",
            "comments_url": "https://api.github.com/repos/nvaccess/nvda/issues/562/comments",
            "events_url": "https://api.github.com/repos/nvaccess/nvda/issues/562/events",
            "html_url": "https://github.com/nvaccess/nvda/issues/562",
            "id": 116022735,
            "number": 562,
            "title": "NVDA stops speaking of typed characters from time to time",
            "user": {
                "login": "nvaccessAuto",
                "id": 14253915,
                "avatar_url": "https://avatars1.githubusercontent.com/u/14253915?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/nvaccessAuto",
                "html_url": "https://github.com/nvaccessAuto",
                "followers_url": "https://api.github.com/users/nvaccessAuto/followers",
                "following_url": "https://api.github.com/users/nvaccessAuto/following{/other_user}",
                "gists_url": "https://api.github.com/users/nvaccessAuto/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/nvaccessAuto/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/nvaccessAuto/subscriptions",
                "organizations_url": "https://api.github.com/users/nvaccessAuto/orgs",
                "repos_url": "https://api.github.com/users/nvaccessAuto/repos",
                "events_url": "https://api.github.com/users/nvaccessAuto/events{/privacy}",
                "received_events_url": "https://api.github.com/users/nvaccessAuto/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 284590813,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/bug",
                    "name": "bug",
                    "color": "fc2929",
                    "default": true
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 18,
            "created_at": "2010-02-16T05:40:42Z",
            "updated_at": "2015-11-10T04:20:24Z",
            "closed_at": null,
            "body": "**Reported by aleksey_s on 2010-02-16 05:40**\nI noticed, that NVDA sometimes stops speaking of typed characters. It happens every time after resuming from standby mode and sometimes without any reason. Restarting of NVDA helps resolve this one. I think this was introduced in Sunday commits.\n",
            "score": 0.37493137
        },
        {
            "url": "https://api.github.com/repos/sympy/sympy/issues/4966",
            "repository_url": "https://api.github.com/repos/sympy/sympy",
            "labels_url": "https://api.github.com/repos/sympy/sympy/issues/4966/labels{/name}",
            "comments_url": "https://api.github.com/repos/sympy/sympy/issues/4966/comments",
            "events_url": "https://api.github.com/repos/sympy/sympy/issues/4966/events",
            "html_url": "https://github.com/sympy/sympy/issues/4966",
            "id": 28950379,
            "number": 4966,
            "title": "plotting tests open Python rocket ship application in Mac OS X",
            "user": {
                "login": "asmeurer",
                "id": 71486,
                "avatar_url": "https://avatars0.githubusercontent.com/u/71486?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/asmeurer",
                "html_url": "https://github.com/asmeurer",
                "followers_url": "https://api.github.com/users/asmeurer/followers",
                "following_url": "https://api.github.com/users/asmeurer/following{/other_user}",
                "gists_url": "https://api.github.com/users/asmeurer/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/asmeurer/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/asmeurer/subscriptions",
                "organizations_url": "https://api.github.com/users/asmeurer/orgs",
                "repos_url": "https://api.github.com/users/asmeurer/repos",
                "events_url": "https://api.github.com/users/asmeurer/events{/privacy}",
                "received_events_url": "https://api.github.com/users/asmeurer/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 84627344,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/bug",
                    "name": "bug",
                    "color": "ededed",
                    "default": true
                },
                {
                    "id": 84627341,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/imported",
                    "name": "imported",
                    "color": "ededed",
                    "default": false
                },
                {
                    "id": 84627346,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/plotting",
                    "name": "plotting",
                    "color": "0000FF",
                    "default": false
                },
                {
                    "id": 84627389,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/Testing",
                    "name": "Testing",
                    "color": "c7def8",
                    "default": false
                },
                {
                    "id": 84627345,
                    "url": "https://api.github.com/repos/sympy/sympy/labels/valid",
                    "name": "valid",
                    "color": "d4c5f9",
                    "default": false
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": {
                "login": "asmeurer",
                "id": 71486,
                "avatar_url": "https://avatars0.githubusercontent.com/u/71486?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/asmeurer",
                "html_url": "https://github.com/asmeurer",
                "followers_url": "https://api.github.com/users/asmeurer/followers",
                "following_url": "https://api.github.com/users/asmeurer/following{/other_user}",
                "gists_url": "https://api.github.com/users/asmeurer/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/asmeurer/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/asmeurer/subscriptions",
                "organizations_url": "https://api.github.com/users/asmeurer/orgs",
                "repos_url": "https://api.github.com/users/asmeurer/repos",
                "events_url": "https://api.github.com/users/asmeurer/events{/privacy}",
                "received_events_url": "https://api.github.com/users/asmeurer/received_events",
                "type": "User",
                "site_admin": false
            },
            "assignees": [
                {
                    "login": "asmeurer",
                    "id": 71486,
                    "avatar_url": "https://avatars0.githubusercontent.com/u/71486?v=3",
                    "gravatar_id": "",
                    "url": "https://api.github.com/users/asmeurer",
                    "html_url": "https://github.com/asmeurer",
                    "followers_url": "https://api.github.com/users/asmeurer/followers",
                    "following_url": "https://api.github.com/users/asmeurer/following{/other_user}",
                    "gists_url": "https://api.github.com/users/asmeurer/gists{/gist_id}",
                    "starred_url": "https://api.github.com/users/asmeurer/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/asmeurer/subscriptions",
                    "organizations_url": "https://api.github.com/users/asmeurer/orgs",
                    "repos_url": "https://api.github.com/users/asmeurer/repos",
                    "events_url": "https://api.github.com/users/asmeurer/events{/privacy}",
                    "received_events_url": "https://api.github.com/users/asmeurer/received_events",
                    "type": "User",
                    "site_admin": false
                }
            ],
            "milestone": null,
            "comments": 6,
            "created_at": "2010-03-20T16:09:09Z",
            "updated_at": "2015-04-17T16:58:12Z",
            "closed_at": null,
            "body": "```\nWhenever I run the test suite on my Mac OS X 10.6 machine, as soon as I get to the plotting tests, \nthe Python rocket ship application (the one in \n/Library/Frameworks/Python.framework/Versions/2.6/Resources/Python.app in the default Mac OS \nX install) starts bouncing in the Dock, and keeps bouncing until the tests complete, at which point it \nquits.  No windows ever open, but it is really annoying nonetheless because it steals the keyboard \nfocus whenever it happens, and if the tests run for long enough and the computer is idle, it causes \nthe \"beachball of death\" to start spinning until you click on something else.  \n\nUse of the -k option reveals that this happens with any test in test_plotting.py, though the program \nlaunches only once per test run.  \n\nDo any Linux or Windows users have this problem?  I can see why it is happening, because plots \nopen in that same rocket ship application, but it is really annoying and if it could somehow not be \nopened or opened in the background, it would be great.\n```\n\nOriginal issue for #4966: http://code.google.com/p/sympy/issues/detail?id=1867\nOriginal author: https://code.google.com/u/asmeurer@gmail.com/\nReferenced issues: #5243\nOriginal owner: https://code.google.com/u/asmeurer@gmail.com/\n",
            "score": 0.381608
        },
        {
            "url": "https://api.github.com/repos/nvaccess/nvda/issues/606",
            "repository_url": "https://api.github.com/repos/nvaccess/nvda",
            "labels_url": "https://api.github.com/repos/nvaccess/nvda/issues/606/labels{/name}",
            "comments_url": "https://api.github.com/repos/nvaccess/nvda/issues/606/comments",
            "events_url": "https://api.github.com/repos/nvaccess/nvda/issues/606/events",
            "html_url": "https://github.com/nvaccess/nvda/issues/606",
            "id": 116022897,
            "number": 606,
            "title": "Inaccessible features treeview in Microsoft Office 2010 custom setup",
            "user": {
                "login": "nvaccessAuto",
                "id": 14253915,
                "avatar_url": "https://avatars1.githubusercontent.com/u/14253915?v=3",
                "gravatar_id": "",
                "url": "https://api.github.com/users/nvaccessAuto",
                "html_url": "https://github.com/nvaccessAuto",
                "followers_url": "https://api.github.com/users/nvaccessAuto/followers",
                "following_url": "https://api.github.com/users/nvaccessAuto/following{/other_user}",
                "gists_url": "https://api.github.com/users/nvaccessAuto/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/nvaccessAuto/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/nvaccessAuto/subscriptions",
                "organizations_url": "https://api.github.com/users/nvaccessAuto/orgs",
                "repos_url": "https://api.github.com/users/nvaccessAuto/repos",
                "events_url": "https://api.github.com/users/nvaccessAuto/events{/privacy}",
                "received_events_url": "https://api.github.com/users/nvaccessAuto/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [
                {
                    "id": 284590813,
                    "url": "https://api.github.com/repos/nvaccess/nvda/labels/bug",
                    "name": "bug",
                    "color": "fc2929",
                    "default": true
                }
            ],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 4,
            "created_at": "2010-03-25T10:14:26Z",
            "updated_at": "2015-11-10T04:22:01Z",
            "closed_at": null,
            "body": "**Reported by Agent Golder on 2010-03-25 10:14**\nI just noticed that in some situations such as the Microsoft Office 2010 custom setup, features treeview can't be read normally but only using object navigation which sometimes is difficult to perform actions.\nWhile of course, with Jaws anything gose normally.\nWhy is this? Can a fix be made?\n",
            "score": 0.28862232
        }
    ]
};
const GitHub = require('github-api');

if (!process.env.GH_TOKEN) {
    throw new Error("process.env.GH_TOKEN should be set GitHub Personal token");
}

export class GitHubClient {
    gh: any;

    constructor(gitHubSetting: GitHubSetting) {
        this.gh = new GitHub({
            token: gitHubSetting.token
        }, gitHubSetting.apiHost);

    }

    search(query: GitHubSearchQuery): Promise<GitHubSearchResult> {
        return this.gh.search().forIssues({
            q: query.query,
            sort: "updated" // always updated
        }).then((response: { config: Object, data: Item[], header: Object, request: any, status: number, statusText: string }) => {
            console.log(response);
            return GitHubSearchResultFactory.create({
                items: response.data
            });
        });
    }
}